pygerduty/events_v2.py

```diff
@@ -0,0 +1,81 @@
+# These methods are compatible with Pagerduty Events API v2
+#  - https://v2.developer.pagerduty.com/docs/events-api-v2
+#  - https://support.pagerduty.com/docs/pd-cef
+
+from six.moves import urllib
+from .exceptions import Error, IntegrationAPIError, BadRequest, NotFound
+from .common import (
+    _json_dumper,
+    Requester,
+)
+
+
+INTEGRATION_API_URL = "https://events.pagerduty.com/v2/enqueue"
+
+
+class Events(object):
+    def __init__(self, routing_key, requester=None):
+        self.routing_key = routing_key
+        self.headers = {
+            "Content-type": "application/json",
+            "Accept": "application/vnd.pagerduty+json;version=2",
+            "X-Routing-Key": self.routing_key
+        }
+        if requester is None:
+            self.requester = Requester()
+        else:
+            self.requester = requester
+
+    def enqueue(self, **kwargs):
+        # Required (according to documentation) PD-CEF fields
+        data = {"event_action": kwargs['event_action']}
+        data['payload'] = kwargs.get('payload', {})
+        for key in ['summary', 'source', 'severity']:
```

Comment:
A teammate pointed out that `payload` is optional, but looks like this assumes payload is required (by checking for these three keys).

Reply:
Thanks for pointing this out. 👍